presto-base-jdbc/src/main/java/io/prestosql/plugin/jdbc/BaseJdbcClient.java

```diff
@@ -241,11 +248,10 @@ public BaseJdbcClient(
         }
     }
 
-    @Override
-    public List<JdbcColumnHandle> getColumns(ConnectorSession session, JdbcTableHandle tableHandle)
+    protected List<JdbcColumnHandle> getColumns(ConnectorSession session, SchemaTableName schemaTableName, Optional<String> catalogName, Optional<String> schemaName, Optional<String> tableName)
```

Comment:
Why do we pass a `schemaTableName` and also a `schemaName` and a `tableName`? What's the difference between them?

Reply:
One is Presto name and other is name in remote database. These could be different.

Reply:
See https://github.com/prestosql/presto/pull/3337

Reply:
Got it. The lack of differentiation between the names is confusing, so I'm all for fixing that.